pkg/controller/node/BUILD

```diff
@@ -38,6 +38,7 @@ go_library(
         "//pkg/cloudprovider:go_default_library",
         "//pkg/cloudprovider/providers/gce:go_default_library",
         "//pkg/controller:go_default_library",
+        "//pkg/kubelet:go_default_library",
```

Comment:
This is shared (duplicated) to implement a heuristic around distributed consistency reasoning regarding the state of the kubelet.  

The reason they are forced to match it is to make sure that the nodecontroller logic  is tightly calibrated to interpret node heart beating failures in a way that is most likely to be "correct". 

 If the value is too large, there is a unnecessarily large delay in responding to zombie conditions . 

If it's too short, then it may be too aggressively interpret an node as being a zombie.

.... tldr I think an integration test will be a way to implement the same reasoning enforcement, and the constants can be duplicated safely that way without having to worry about regression in the behaviour.

.....

So would you prefer (1) a kubelet/constants package or a top level constants package or (2) an integration test ?  Either is feasible as an alternative to a hard import.